### PR TITLE
feat: added admin password reset procedure

### PR DESCRIPTION
**Pull request by Ani1357**

### CODE REVIEW

File: docs/for-ops/how-to/change-admin-password.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+3. Create the `.secrets` file:
+
+- Get the SOPS AGE KEY secret:
+
+```
+kubectl get secret otomi-sops-secrets -n otomi-pipelines -o jsonpath='{.data.SOPS_AGE_KEY}' | base64 -d
+```
+
+- Create the `.secrets` file in the root of the values directory with the SOPS_AGE_KEY secret. The file's content should look like this:
```
Comment (CasLubbers) [inline docs/for-ops/how-to/change-admin-password.md:67]:
```suggestion
- Create the `.secrets` file in the root of the values directory with the SOPS_AGE_KEY secret. The file contents should look like this:
```

File: docs/for-ops/how-to/change-admin-password.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+```bash
+cd ~/workspace/values-folder
+git add .
+git commit -m "Update otomi-admin password"
+git push
+```
+
+### Step 5: Apply the Changes
+
+1. Allow the tekton pipeline to run and verify it passes.
```
Comment (CasLubbers) [inline docs/for-ops/how-to/change-admin-password.md:105]:
```suggestion
1. Allow the Tekton pipeline to run and verify it passes.
```

File: docs/for-ops/how-to/change-admin-password.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+2. After the pipeline completes, restart the `otomi-api` and `apl-keycloak-operator` to ensure it applies the new credentials.
+    
+    ```bash
+    kubectl rollout restart deployment -n otomi otomi-api
+    kubectl rollout restart deployment -n apl-keycloak-operator apl-keycloak-operator
+    ```
+
+3. Confirm that the Otomi platform is working as expected with the new credentials.
```
Comment (CasLubbers) [inline docs/for-ops/how-to/change-admin-password.md:114]:
```suggestion
3. Verify that the Otomi platform is working as expected with the new credentials.
```

File: docs/for-ops/how-to/change-user-passwords.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+sidebar_label: Change user passwords
+---
+
+Follow these steps to change or reset passwords of users created in the Console or the `platform-admin` account created during installation:
+
+1. Open the Keycloak app
+
+2. Click on `Administration Console`.
+
+3. Login with `otomi-admin` credentials.
```
Comment (CasLubbers) [inline docs/for-ops/how-to/change-user-passwords.md:13]:
```suggestion
3. Log in with `otomi-admin` credentials.
```

**Review by CasLubbers — APPROVED** [review]
Some minor adjustments overal looks good.

File: docs/for-ops/how-to/change-admin-password.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+3. SOPS set up with Age encryption (default in LKE).
+
+## Steps:
+
+### Step 1: Generate a New Password
+
+```bash
+    # Generate a random 16-character alphanumeric password
+    head /dev/urandom | tr -dc A-Za-z0-9 | head -c 16
```
Comment (CasLubbers) [inline docs/for-ops/how-to/change-admin-password.md:23]: This command doesn't work for me
`tr: Illegal byte sequence`
You can also say generate a uuid? Which can be done with:
`uuidgen`

**Review by CasLubbers — COMMENTED** [review]
One comment but the rest worked

File: docs/for-ops/how-to/change-admin-password.md (hunk 5 above)
Comment (Ani1357) [reply docs/for-ops/how-to/change-admin-password.md]: Updated existing command. Not it should work with MacOS coreutils.

**Pull request merged**